utils/lambda-creator.js

```diff
@@ -74,7 +74,7 @@ exports.createLambda = async (
             Role: iamRole.arn, // IAM_ROLE_ARN; e.g., arn:aws:iam::650138640062:role/v3-lambda-tutorial-lambda-role
             Runtime: 'nodejs16.x',
             Description: description,
-            MemorySize: 512,
+            MemorySize: 1024,
```

Comment:
Do you think maybe we should make it configurable from the user's side, with a default value of 1024? The same goes for the timeout parameter.